qt.cmake

```diff
@@ -29,6 +29,12 @@ else()
 		message(FATAL_ERROR "Failed to determine Qt flavour")
 	endif()
 
+	# Debug suffix
```

Comment:
Isn't there a need to set this flag for debug configuration only?
Something like that:
```
if (CMAKE_BUILD_TYPE EQUAL "DEBUG")
    # set suffix, etc.
endif (CMAKE_BUILD_TYPE EQUAL "DEBUG")
```

Reply:
Why there's need to set this flag for debug configuration only if we're configuring library interface? What would be the benefit of configuring this only for debug while library interface is using this variable in all build-types?